scripts/README.md

```diff
@@ -1,51 +1,47 @@
+	- `num_lines`: number of lines displayed from the end of each log file
 
-[TODO: ADD ARGUMENT OPTIONS]
+- `SearchAlgorithmTuning_V1.ipynb`: This is a jupyter notebook used for prototyping and testing the search system.
 
-- `script_warnings'
-    - `searchlog_warnings.txt': This records stderr output piped from the `run_search.py' script
-    - `classlog_warnings.txt': This records stderr output piped from the `run_classifier.py' script
+- `get_status.sh`: This function reads log files currently in the `dsa110-nsfrb/tmpoutput` folder and outputs them to terminal for periodic monitoring, taking the wait time between updates in seconds as the argument
 
-- `script_flags'
-    - `searchlog_flags.txt': This records data info about data piped from `run_search.py' to `run_classifier.py' needed to convert raw hex to numpy array. This includes `datasize' (int specifying the total number of bytes output from `run_search.py'), `outputshape' (tuple specifying the shape of the numpy array), and `size' (int specifying the number of bits for each entry in the numpy array); each argument is delimited by a `;'.
-
-- `get_status.sh': This function reads log files currently in the `dsa110-nsfrb/tmpoutput' folder and outputs them to terminal for periodic monitoring, taking the wait time between updates in seconds as the argument
-
-- `socket_client_test_PUT.sh': This script uses curl (https://curl.se/) to take an input datafile and send using an http PUT command to the T4 server. 
-- `socket_client_test_POST.sh' [deprecated] : This script uses curl (https://curl.se/) to take an input datafile and send using an http POST command to the T4 server. This script has been replaced by the more reliable and efficient `socket_client_test_PUT.sh'.
+- `socket_client_test_PUT.sh`: This script uses curl (https://curl.se/) to take an input datafile and send using an http PUT command to the T4 server. 
+- `socket_client_test_POST.sh` [deprecated] : This script uses curl (https://curl.se/) to take an input datafile and send using an http POST command to the T4 server. This script has been replaced by the more reliable and efficient `socket_client_test_PUT.sh`.
 
+- `test_search.py`: This script runs unit tests for the search pipeline.
 
 ## Usage
 
-To initiate the search and classification pipeline, run the following commands:
+To run the imaging test script:
 
 ```bash
-cd /home/ubuntu/proj/dsa110-shell/dsa110-nsfrb/scripts
-./run_search | ./run_classifier
+python ms_vis_extraction.py [num_gulp] [num_time_samples]
+```
 
-To monitor the output, open a separated terminal and run:
-```bash
-cd /home/ubuntu/proj/dsa110-shell/dsa110-nsfrb/scripts
-./get_status.sh [time]
+To monitor the log files:
 
-where [time] is the time between status updates in seconds (recommend 10-15 seconds). Stop the monitor with Ctrl+c
+```bash
+./get_status.sh [wait_time] [num_lines]
+```
 
-To send data using the client script, run:
+To run all unit tests for the search pipeline with 300x300 images:
```

Comment:
@mbsherma remove this b/c testing moved to tests folder

Reply:
Resolved in commit [78cb0ed](https://github.com/dsa110/dsa110-nsfrb/pull/10/commits/78cb0edfc85ffdf445089667f71141e579d956f9)